tests/upload_test.go

```diff
@@ -99,21 +141,24 @@ var _ = Describe("[rfe_id:138][crit:high][vendor:cnv-qe@redhat.com][level:compon
 			Expect(err).ToNot(HaveOccurred())
 			Expect(found).To(BeTrue())
 
-			By("Verify content")
-			same, err := f.VerifyTargetPVCContentMD5(f.Namespace, pvc, utils.DefaultImagePath, utils.UploadFileMD5, false)
-			Expect(err).ToNot(HaveOccurred())
-			Expect(same).To(BeTrue())
-			fileSize, err := f.RunCommandAndCaptureOutput(pvc, "stat -c \"%s\" /pvc/disk.img")
-			Expect(err).ToNot(HaveOccurred())
-			Expect(fileSize).To(Equal("1073741824")) // 1G
+			if !isBlockVolume {
+				By("Verify content")
+				same, err := f.VerifyTargetPVCContentMD5(f.Namespace, pvc, utils.DefaultImagePath, utils.UploadFileMD5, false)
+				Expect(err).ToNot(HaveOccurred())
+				Expect(same).To(BeTrue())
+				fileSize, err := f.RunCommandAndCaptureOutput(pvc, "stat -c \"%s\" /pvc/disk.img")
+				Expect(err).ToNot(HaveOccurred())
+				Expect(fileSize).To(Equal("1073741824")) // 1G
+			}
 		} else {
 			By("Verify PVC empty")
 			_, err = framework.VerifyPVCIsEmpty(f, pvc)
 			Expect(err).ToNot(HaveOccurred())
 		}
 	},
-		table.Entry("[test_id:1368]succeed given a valid token", true, http.StatusOK),
-		table.Entry("[posneg:negative][test_id:1369]fail given an invalid token", false, http.StatusUnauthorized),
+		table.Entry("[test_id:1368]succeed given a valid token", true, http.StatusOK, false),
+		table.Entry("[test_id:1368]succeed given a valid token", true, http.StatusOK, true),
```

Comment:
Maybe remove the test_id so that QE will update accordingly

Reply:
yeah. forgot... 
the first line is an old one. just added a new parameter.